src/overlays/actors/ovl_Dm_Sa/z_dm_sa.h

```diff
@@ -10,13 +10,11 @@ typedef void (*DmSaActionFunc)(struct DmSa*, PlayState*);
 typedef struct DmSa {
     /* 0x000 */ Actor actor;
     /* 0x144 */ SkelAnime skelAnime;
-    /* 0x188 */ UNK_TYPE1 unk194[0x108];
+    /* 0x188 */ UNK_TYPE1 unk_188[0x108];
     /* 0x290 */ DmSaActionFunc actionFunc;
-    /* 0x294 */ UNK_TYPE1 unk294[0x20];
-    /* 0x2B4 */ s32 unk2B4;
-    /* 0x2B8 */ UNK_TYPE1 unk2B8[0x28];
-    /* 0x2E0 */ u16 unk2E0;
-    /* 0x2E2 */ UNK_TYPE1 unk2E2[0xE];
+    /* 0x294 */ UNK_TYPE1 unk_294[0x4C];
+    /* 0x2E0 */ u16 unk_2E0;
+    /* 0x2E2 */ UNK_TYPE1 unk_2E2[0xE];
```

Comment:
Feels a little weird to infer this in the actor that doesnt use 2E0 just because it matches the other, but I guess it's fine?